tensorflow/contrib/tensorrt/convert/convert_nodes.cc

```diff
@@ -1301,7 +1301,6 @@ tensorflow::Status ConvertConv2DHelper(
 
   layer->setStride(stride);
   layer->setPadding({padding[0].first, padding[1].first});
-  layer->setName(node_def.name().c_str());
```

Comment:
Now that this is removed, where are the names set?

Reply:
I think we don't need the names, see my comment above.

Reply:
@aaroey if possible we would like to keep the layer names and tensor names since they are used in TRT log messages and help us pinpoint issues more easily.